epub33/core/index.html

```diff
@@ -8711,9 +8711,49 @@ <h2>Security and Privacy</h2>
 		<section id="app-overview-unsupported" class="appendix">
 			<h2>Unsupported Features</h2>
 
-			<p>This specification contains certain features the Working Group no longer recommends for use or that are
-				only retained for legacy support reasons. This section defines the meanings of the designations attached
-				to these features and their support expectations.</p>
+			<p>This specification contains certain features that are not yet fully supported in Reading Systems, that
+				the Working Group no longer recommends for use, or that are only retained for interoperability with EPUB
+				2 Reading Systems. This section defines the meanings of the designations attached to these features and
+				their support expectations.</p>
+
+			<section id="risky">
+				<h3>Risky Features</h3>
+
+				<p>A <strong>risky</strong> feature is a feature introduced prior to EPUB 3.3 but that the Working Group
```

Comment:
"risky" sounds odd to me, and also to close to "at risk".
But I do not have a better suggestion 😅

Reply:
Other than beta, the only other thought I've had is "unproven". Technically, that is what we're saying - we haven't been able to prove the features have adoption.

Their being "risky" seems to depend on how the future unfolds. Some could stabilize not long after we publish. (At least in theory, not yet knowing what all will get labelled.)

Reply:
So far, "unproven" is at the top of my list...

Reply:
Another term I dreamt up (literally, I thought of it at dawn today) is "incubated".

Reply:
We should probably merge this PR without closing 1944 and take the debate on the right name back to that issue.